### PR TITLE
Update example-intro to use the screen function

### LINKED ISSUE #395: Examples In Docs Site Use Render Destructuring Instead of `screen` Import
**Describe the bug**
As mentioned in the conversation regarding ReactTraining/react-router#7169 -- the README is up to date with the current guidance on how to use `screen` instead of destructuring the `render` function.

**To Reproduce**
1. Go to https://testing-library.com/docs/react-testing-library/example-intro
2. See that the render function is being destructured to access get functions.

**Expected behavior**
The docs are updated to show `screen` being used instead.

### PR DESCRIPTION
**Pull request by thatzacdavis**
This should resolve #395

### CODE REVIEW

File: docs/react-testing-library/example-intro.md [hunk 1]
```diff
@@ -20,20 +20,20 @@ jest.mock('axios')
 
 test('loads and displays greeting', async () => {
   const url = '/greeting'
-  const { getByText, getByRole } = render(<Fetch url={url} />)
+  const { container, asFragment } = render(<Fetch url={url} />)
```
Comment (eps1lon) [inline docs/react-testing-library/example-intro.md]: Looks like c&p from below or why do we need these now?

Comment (thatzacdavis) [reply docs/react-testing-library/example-intro.md]: Oh yes, it definitely was. I'll remove them.

**Comment by alexkrolick** [comment]
Merging despite Netlify's lack of input

**Comment by eps1lon** [comment]
> Merging despite Netlify's lack of input

Should be resolved (see https://community.netlify.com/t/flaky-github-status-check-reporting/10817).

**Pull request merged**